src/brpc/rdma/rdma_endpoint.cpp

```diff
@@ -972,9 +1001,9 @@ ssize_t RdmaEndpoint::HandleCompletion(ibv_wc& wc) {
 
             // Update window
             uint32_t wnd_thresh = _local_window_capacity / 8;
-            if (_window_size.fetch_add(acks, butil::memory_order_relaxed) >= wnd_thresh
+            if (_remote_rq_window_size.fetch_add(acks, butil::memory_order_relaxed) >= wnd_thresh
```

Comment:
This part receives an update on the rq queue size from Recv, so it would be more appropriate to call this variable _rq_window_size, which corresponds to _sq_window_size mentioned above.